diff/parse.go

```diff
@@ -247,22 +250,21 @@ func (r *FileDiffReader) HunksReader() *HunksReader {
 // start with "---" and "+++" with the orig/new file names and
 // timestamps). Or which starts with "Only in " with dir path and filename.
 // "Only in" message is supported in POSIX locale: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/diff.html#tag_20_34_10
-func (r *FileDiffReader) ReadFileHeaders() (origName, newName string, origTimestamp, newTimestamp *time.Time, err error) {
+func (r *FileDiffReader) ReadFileHeaders() (origName, newName string, origTimestamp, newTimestamp *time.Time, origTZ, newTZ bool, err error) {
```

Comment:
this is a backwards incompatible change to the API, which will break users of this library.

The reason you are tracking this info is so you can re-marshal the diff byte for byte identically? Is that important? I would consider when we marshal back out we just marshal in UTC with tz.

Reply:
I wasn't sure if it was important or not. I'll make that change and adjust the tests tomorrow.